### PR TITLE
Auto increment copyright and specify version

### LINKED ISSUE #70: Add automated versioning number to docs
The documentation could show the version number directly from the git commit/label. The developer guide should point to `latest` where most of the user documentation should direct to the released documentation.

### PR DESCRIPTION
**Pull request by paddyroddy**
Fixes #70. 

I need to rely on CI as I can't install `cvxopt` 😢

### CODE REVIEW

**Comment by paddyroddy** [comment]
@p-j-smith I can't deploy this locally due to `cvxopt` problems 😢 does this look right if you build it?

File: docs/conf.py [hunk 1]
```diff
@@ -20,12 +21,13 @@
 # -- Project information -----------------------------------------------------
 
 project = "Bayesian Tracker (btrack) 🔬💻"
-copyright = "2022, Alan R Lowe"
 author = "Alan R Lowe"
+copyright = f"2022-{datetime.date.today().year}, {author}"
```
Comment (p-j-smith) [inline docs/conf.py:25]: do you need to add the current year? Having just the initial year is okay too, or even no year

File: docs/conf.py [hunk 2]
```diff
@@ -20,12 +21,13 @@
...
 
 
 # The full version, including alpha/beta/rc tags
-release = "v" + btrack.__version__
+version = f"v{btrack.__version__}"
+release = version
```
Comment (p-j-smith) [inline docs/conf.py:30]: is `version` needed here? could you do:
```python
release = f"v{btrack.__version__}"
```

**Review by p-j-smith — COMMENTED** [review]
just a couple of questions. It builds okay locally with tox

File: docs/conf.py (hunk 2 above)
Comment (paddyroddy) [reply docs/conf.py:30]: https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information

> If you don’t need the separation provided between [version](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version) and [release](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-release), just set them both to the same value.

I took this to mean you should have both

**Comment by paddyroddy** [comment]
> just a couple of questions. It builds okay locally with tox

I meant go in `docs` and run `make`. I want to preview the HTML.

File: docs/conf.py (hunk 2 above)
Comment (p-j-smith) [reply docs/conf.py:30]: yeah that does sounds like both are needed

File: docs/conf.py (hunk 1 above)
Comment (p-j-smith) [reply docs/conf.py:25]: a range is probably fine - at least it's automated. But is 2022 the correct start year? Or is it an outdated 'current' year?

Comment (paddyroddy) [reply docs/conf.py:25]: Yeah I'd argue 2022 seems too late, but I don't know when `btrack` started

Comment (p-j-smith) [reply docs/conf.py:25]: the license file is from 2018, but may be even earlier?

**Comment by p-j-smith** [comment]
> I want to preview the HTML.

to check the version is displayed correctly? It shows `v0.5.0`. It should ideally show `v0.6.0dev`, but that's a different issue

**Comment by paddyroddy** [comment]
> > I want to preview the HTML.
> 
> to check the version is displayed correctly? The copyright years are correct and it shows `v0.5.0`. It should ideally show `v0.6.0dev`, but that's a different issue

Great, thanks, that's what I wanted. Yes, that is related to #219.

File: docs/conf.py (hunk 1 above)
Comment (p-j-smith) [reply docs/conf.py:25]: looking at the [README history](https://github.com/quantumjot/BayesianTracker/commit/ece143930813b135578766d846af81505f5ce172) the starting year should be 2017

Comment (paddyroddy) [reply docs/conf.py:25]: Okay I'll add that for now

**Review by quantumjot — APPROVED** [review]

**Pull request merged**